[ENH] run GLM on surface data on BIDS datasets (#4507)
nilearn/nilearn: doc/changes/latest.rst

- :bdg-dark:`Code` :func:`nilearn.image.binarize_img` explicitly cast images to ``int8`` to avoid warnings about ``int64`` when working with ``float64`` images (:gh:`4498` by `Patrick Sadil`_).

> - :bdg-dark:`Code` :func:`nilearn.glm.first_level.first_level_from_bids` will for images in space 

Author: ```suggestion
- :bdg-dark:`Code` :func:`nilearn.glm.first_level.first_level_from_bids` will for images in MNI space 
```

> `` if not space is provided (:gh:`4507` by `Rémi Gau`_).

Author: ```suggestion
if no space is provided (:gh:`4507` by `Rémi Gau`_).
```

> - :bdg-dark:`Code` :func:`nilearn.glm.first_level.first_level_from_bids` willm for images in space ``- :bdg-dark:`Code` Improve :func:`nilearn.glm.first_level.first_level_from_bids` to look for and load ``fsaverage5`` data to easily run GLM on surface data (:gh:`4507` by `Rémi Gau`_).

Author: ```suggestion
- :bdg-dark:`Code` :func:`nilearn.glm.first_level.first_level_from_bids` will for images in space 
```
